components/encryption/src/encrypted_file/mod.rs

```diff
@@ -69,8 +69,7 @@ impl<'a> EncryptedFile<'a> {
         let mut tmp_file = OpenOptions::new()
             .create(true)
             .write(true)
-            .open(&tmp_path)
-            .unwrap_or_else(|e| panic!("EncryptedFile::write {:?}: {}", &tmp_path.to_str(), e));
+            .open(&tmp_path)?;
```

Comment:
It looks like we have lost a very useful error message.

Reply:
Good point, let me send a new PR to log the path.